test.sh

```diff
@@ -36,9 +36,20 @@ start schema
 ./src/schemas/schema-schema.sh
 end schema
 
+BUILD_FILES=(
+  es/production/index.min.js
+  es/production/static/css/index.css
+  es/development/index.js
+  es/development/static/css/index.css
+  umd/production/index.min.js
+  umd/production/static/css/index.css
+  umd/development/index.js
+  umd/development/static/css/index.css
+)
+
 start build
 npm run build
-for F in es/index.es.js umd/index.umd.js es/static/css/main.css umd/static/css/main.css; do
+for F in ${BUILD_FILES[*]}; do
```

Comment:
I was wondering this too. I think we may want to check that the small files are exported since if these are all pushed to NPM then people may begin to depend on them. I will update this.

Reply:
I have added a small script to do this here https://github.com/hubmapconsortium/vitessce/pull/513/commits/cd9d4f6182d1067969930603127fe8cd67af90d4